scripts/prepare_dolly.py

```diff
@@ -0,0 +1,168 @@
+"""Implementation derived from https://github.com/tloen/alpaca-lora"""
+import json
+import sys
+from pathlib import Path
+
+import requests
+import torch
+from torch.utils.data import random_split
+from tqdm import tqdm
+
+# support running without installing as a package
+wd = Path(__file__).parent.parent.resolve()
+sys.path.append(str(wd))
+
+from lit_gpt.tokenizer import Tokenizer
+
+DATA_FILE_URL = (
+    "https://huggingface.co/datasets/databricks/databricks-dolly-15k/resolve/main/databricks-dolly-15k.jsonl"
+)
+DATA_FILE_NAME = "dolly_data_cleaned.json"
+DESTINATION_PATH = Path("data/dolly")
+CHECKPOINT_DIR = Path("checkpoints/stabilityai/stablelm-base-alpha-3b")
+TEST_SPLIT_FRACTION = 0.1333  # to get exactly 2000 test samples
```

Comment:
We were setting exactly 2k test samples for compatibility with previous experiments, but in this case we could do 10% directly

```suggestion
TEST_SPLIT_FRACTION = 0.1
```